### PR TITLE
ui/settings/network.ui: clarify autoaway pref use

### PR DESCRIPTION
**Pull request by AtticFinder65536**
To keep backwards compatibility, the previous behaviour that autoway is disabled when set to 0 should be kept, so no need to make it a button, just clarify the existing behaviour in the UI.

### CODE REVIEW

**Comment by mathiascode** [comment]
Keep in mind that strings should generally be short. Your string spans two lines, and will likely span three in some translations, as translated strings are typically longer.

I propose something like this:
`Minutes of inactivity until away (0 to disable):`

If anyone has better suggestions around that length, feel free to mention them.

**Comment by slook** [comment]
I too noticed that the string was long, consider a ToolTip if an expanded description is required.

File: pynicotine/gtkgui/ui/settings/network.ui [hunk 1]
```diff
@@ -251,7 +251,8 @@
                   <object class="GtkLabel">
                     <property name="visible">True</property>
                     <property name="hexpand">True</property>
-                    <property name="label" translatable="yes">Toggle away status after minutes of inactivity:</property>
+                    <property name="label" translatable="yes">Minutes of inactivity before going away (0 to disable):</property>
+                    <property name="tooltip-text" translatable="yes">Once you interact with the application after idling, status will be set to online.</property>
```
Comment (slook) [inline pynicotine/gtkgui/ui/settings/network.ui:255]: "idling" -> "being away"

File: pynicotine/gtkgui/ui/settings/network.ui [hunk 2]
```diff
@@ -251,7 +251,8 @@
                   <object class="GtkLabel">
                     <property name="visible">True</property>
                     <property name="hexpand">True</property>
-                    <property name="label" translatable="yes">Toggle away status after minutes of inactivity:</property>
+                    <property name="label" translatable="yes">Minutes of inactivity before going away (0 to disable):</property>
+                    <property name="tooltip-text" translatable="yes">Once you interact with the application being away, status will be set to online.</property>
```
Comment (slook) [inline pynicotine/gtkgui/ui/settings/network.ui:255]: "being away" -> "after being away"

Comment (AtticFinder65536) [reply pynicotine/gtkgui/ui/settings/network.ui:255]: I should get some more sleep.

**Pull request merged**